configuration/white_labels/mo.py

```
@@ -0,0 +1,1450 @@
+    # ==========================================================================================
+    # REFERRER DATA
+    # ==========================================================================================
+
+    # MO uses generic MFB branding and the default step directory, so it inherits
+    # everything from base and overrides only the values that are MO-specific:
+    # the share link and the state name.
+    referrer_data = {
+        **ConfigurationData.referrer_data,
+        "shareLink": {"default": "https://screener.myfriendben.org/mo/step-1"},
+        "stateName": {"default": "Missouri"},
+    }
```

Ruff (0.16.0)

[warning] Mutable default value for class attribute

(RUF012)